lib/src/widgets/yaru_info.dart

```diff
@@ -0,0 +1,118 @@
+import 'package:flutter/material.dart';
+import 'package:yaru/yaru.dart';
+
+enum YaruInfoType {
+  note,
+  tip,
```

Comment:
Maybe success or something? Tip I would expect to be blue

Reply:
I wouldnt limit the enum names to things we currently do
positive green things can be used for other things as a success

eventually instead of tip or success WDYT about suggestion ?

Reply:
I'm not thinking that it limits it to what content the user puts in there, I'm thinking that if a user sees the four basic blocks `success`, `info`, `warning` and `error` (sometimes warning and error are bundled into one) they know what they can expect in terms of icon and color since this is a quite used design pattern. The purple is a newer thing so there doesn't seem to be any term that is "the most popular" for that, I've seen `discovery` for that one too for example, but I think `important` is fine there.

W3 uses similar terms:
https://www.w3schools.com/w3css/w3css_alerts.asp

And bootstrap too:
https://www.w3schools.com/bootstrap/bootstrap_alerts.asp

Atlassian:
![image](https://github.com/ubuntu/yaru.dart/assets/744771/bf95dd3d-a452-4e65-b4fb-1358fff7c29e)

`danger` is a quite common one instead of `error` too.

Reply:
Yes, I agree that `info`, `success`, `warning` and `error`, are the four alert types I would expect to see.

Reply:
okay maybe the checkmark is a bit misleading

I have put the checkmark icon in there as a replacement until we have a light bulb :)

I can change it to information/info, but then maybe instead of 
YaruInfo.info
how about
YaruNote.info
?

danger is def. better than caution because it is stronger and leaves room for other things as errors

I am not really agreeing on the success thing.
But since you both agree I will do it 😄

Reply:
I think merging the success thing with the checkmark would be bad
Any better suggestion as a replacement for the light bulb @Jupi007 ? Otherwise I can just leave it open until then, and maybe only merge the YaruTranslucentBorderContainer first?

Reply:
Maybe we could call it `Alert`, instead if you don't like the repetition of words? :thinking: 

> I have put the checkmark icon in there as a replacement until we have a light bulb :)

Lightbulb I wouldn't necessarily expect on a green background, I think you are looking too hard on GitHub's design, which doesn't necessarily reflect what mobile developers would expect. :wink: 

I guess we'd want a possible override of the icon here in the long run too.

Reply:
What about `YaruAlert`, alert is the common name on the web for that kind of component?

I would see these values:
- info: ℹ️ icon, blue ;
- tip: 💡 icon, blue ;
- success: ✅ icon, green ;
- warning: ⚠️ icon, yellow/orange ;
- important: "!" icon, purple ;
- error: "danger" icon, red.

I'm working on the light bulb icon btw:

![image](https://github.com/ubuntu/yaru.dart/assets/36476595/ad916887-fd9b-4929-876e-5c9106492325)

Reply:
> I think merging the success thing with the checkmark would be bad

How come?
Almost everyone uses a checkmark to indicate success:

![image](https://github.com/ubuntu/yaru.dart/assets/744771/275b984e-95cc-4624-9b0b-51ad9f7e9aa9)

Reply:
Nice icon!

Well, neither mobile nor web should be our reference IMO. As we primarily target the desktop, even though it also works on the web great. And for the desktop our main reference is the GNOME design, even though we leave options to disagree them and put our own ideas/improvements in there. But this is eventually a bit meta/off-topic ;)

Generally it is nice to have more options. As there is already an   final Icon? icon; parameter I guess we are safe

I can leave it open until you have the light bulb done, and add the blue tip enum/box/badge after that. Then all combinations are possible but we have a set of default options we can agree on?

Reply:
@spydon it's okay :) If not forced from other conditions we normally just follow democraity here and since you both agree on the checkmark I will do it. I just think it is great to have those "hey what about this idea?" boxes. I can parameterize it so all options are there but the default is what you too agree on

Reply:
> What about YaruAlert, alert is the common name on the web for that kind of component.

Haha it must indicate a good naming if both of us came up with it separately. :smile: 

> I just think it is great to have those "hey what about this idea?" boxes.

If using @Jupi007's suggestion you'd have that too (not green though) :slightly_smiling_face: 

> tip: 💡 icon, blue ;

Reply:
Okay so what is the plan now?
YaruInfoType/Box/Badge
YaruAlertType/Box/Badge
YaruNoteType/Box/Badge
?
and 

.info
or
.information
?

Reply:
For the enum `YaruAlertType` I think is the best, and `YaruAlertBox` I think is fine.
The badge however, I named that pretty badily to start with, maybe just `YaruBadge` would be enough there?

Reply:
hm I think I have a rather strong favor of YaruInfo
As an information is rather "neutral" colored. Warning information, success information, danger/error information

I can live with YaruInfoType.information if you can too :) Better than the rather negative "alert" IMO

Reply:
cool!
I just wrap the icon Widget parameter in a

```dart
IconTheme(
    data: IconTheme.of(context).copyWith(
      size: 30,
      color: baseColor,
    ),
    child: icon ?? Icon(yaruInfoType.iconData),
  )
```

and with a final baseColor = color ?? yaruInfoType.getColor(context); optional color parameter

then I think we don't need just a special enum for the light bulb thing :) 
As I could just use the light bulb icon

Because double blue is kinda weird?

Now we even feel the colors 😆 We should add how they smell in the docs :D (little joke)

Reply:
I don't think that `alert` sounds bad, but maybe it is because I'm used to this term :p
I'm fine with `YaruInfo` too (I prefer `Alert` though).

Reply:
https://github.com/ubuntu/yaru.dart/assets/15329494/7c8cf3a4-bda6-49f0-9590-92a4d6e3421e

xD

Any further ideas? Otherwise I finish the docs :)

Reply:
Little nitpick, but there's no "danger" badge at the end of the page.

Reply:
Thanks, and you both wanted "error" instead of "danger", right?

Reply:
Honestly I don't care, both sounds good to me :p

Reply:
I'm fine with both of those options!